data/js/freq.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function(){
+    $("#high").click(function(){
+      if ($(this).is(':checked')){
+        $("li.high").fadeIn('slow');
+      } else {$("li.high").fadeOut('slow');}
+    });
+    $("#mid").click(function(){
+      if ($(this).is(':checked')){
+        $("li.mid").fadeIn('slow');
+      } else {$("li.mid").fadeOut('slow');}
+    });
+    $("#low").click(function(){
```

Comment:
Looks like there are 3 almost identical functions which differ only by one keyword ('high', 'mid', and 'low').
Is it possible to merge them into one and make this a parameter?